src/content/documentation/publish/add-on-policies.md

```diff
@@ -127,15 +127,14 @@ In general, developers are free to maintain their add-ons in the manner they cho
 
 The following requirements are of particular importance:
 
-
 - Add-ons must only request those permissions that are necessary for them to function.
 - Add-ons must be self-contained and not load remote code for execution.
 - Add-ons must not load or redirect to a remote new tab page. The new tab page must be contained within the add-on.
 - Add-ons must not relax web page security headers, such as the Content Security Policy.
 - Add-ons must use encryption when transporting data remotely.
 - Add-ons should avoid including redundant code or files.
 - Add-ons must not negatively impact the performance or stability of Firefox.
-- Only release versions of third-party libraries and/or frameworks may be included with an add-on. Modifications to these libraries/frameworks are not permitted. Please read our [third party library guidelines](/documentation/publish/third-party-library-usage/) to better understand related requirements.
+- Only release versions of third-party libraries and/or frameworks may be included with an add-on. Modifications to these libraries/frameworks are not permitted. Please read our [third party library guidelines](/documentation/publish/third-party-library-usage/) to better understand related requirements. {% comment %}Changes to this line should also be copied to /src/content/documentation/publish/third-party-library-usage.md{% endcomment %}
```

Comment:
```suggestion
- Only release versions of third-party libraries and frameworks can be included in an add-on. Modifications to included libraries and frameworks are not permitted. See the [third-party library guidelines](/documentation/publish/third-party-library-usage/) for more information. {% comment %}Changes to this line should also be copied to /src/content/documentation/publish/third-party-library-usage.md{% endcomment %}
```

Reply:
Let's avoid changes to the policies page. Also, if we avoid duplicating the sentence as I suggested in my other comment, the comment is not needed.

Reply:
@rebloor, I'm closing this suggestion because I'm going to revert the change to the policy page per @wagnerand's [request](https://github.com/mozilla/extension-workshop/pull/1918/files#r1686573033).